docs/resource-policy/installation.md

```diff
@@ -1 +1,140 @@
+
+#### Balloons
+
+| Name               | Default                                                                                                                                    | Description                                          |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------- |
+| `namespace`        | kube-system                                                                                                                                | specify the namespace to use for deploying DaemonSet, ConfigMap, CustomResourceDefinition, and RBAC-related objects                                                                                                                                                            |
+| `image.name`       | [ghcr.io/containers/nri-plugins/nri-resource-policy-balloons](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)                 | container image name                                 |
+| `image.tag`        | unstable                                                                                                                                   | container image tag                                  |
+| `image.pullPolicy` | Always                                                                                                                                     | image pull policy                                    |
+| `quota.cpu`        | 500m                                                                                                                                       | cpu qouta for the Pod                                |
+| `quota.memory`     | 512Mi                                                                                                                                      | Memory qouta for the Pod                             |
+| `metrics.port`     | 8891                                                                                                                                       | metrics port on the container                        |
+| `metrics.hostPort` | 8891                                                                                                                                       | metrics port to expose on the host                   |
+| `config`           | <pre><code>Active: topology-aware</code><br><code>ReservedResources:</code><br><code>  cpu: 750m</code></pre>                                                                                                                                                                                                                                                           | plugin configuration data                            |
+
+## Manual installation
```

Comment:
Do we need to have instructions using git clone + make + kubectl apply, or is it too low level and for developers only?

Reply:
or do you mean to say have a Makefile target to run kubectl command?

Reply:
No, I did not mean that. I was just wondering that if user does not have helm or kustomize, what he/she could do then. But that is probably something only for developers and we do not need to think about that here.

Reply:
if no helm or kustommize, they can just simply apply the deployment with kubectl